src/app.js

```diff
@@ -13,12 +13,19 @@ app.use(express.urlencoded({extended: false}));
 app.use(express.json());
 app.use(require('./routers/account'));
 app.use(require('./routers/login'));
+app.use(require('./routers/city'));
+app.use(require('./routers/state'));
+app.use(require('./routers/report'));
+app.use(require('./routers/discussion'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
app.use(require("./routers/discussion"));
```